packages/suite/src/support/messages.ts

```diff
@@ -8730,6 +8731,10 @@ const messages = defineMessagesWithTypeCheck({
         defaultMessage:
             'Entering the staking pool may take up to {count, plural, one {# day} other {# days}}',
     },
+    TR_STAKE_ACTIVATION_COULD_TAKE: {
+        id: 'TR_STAKE_ACTIVATION_COULD_TAKE',
+        defaultMessage: 'Stake activation could take up to 1 epoch (~3 days)',
```

Comment:
technically this may take more epochs if a lot of stake is warming up or cooling down. 
```suggestion
        defaultMessage: 'Stake activation usually takes 1 epoch (~3 days)',
```

Reply:
changed here https://github.com/trezor/trezor-suite/pull/15409/commits/45e2808344e49c757330ec18a4066252f1d98fc1